web/src/pages/index.tsx

```diff
@@ -43,9 +50,9 @@ function scroll30() {
 
 const Home: NextPage = () => {
   const [entries, setEntries] = useState<Entry[]>([]);
-  const [runningIndex, setRunningIndex] = useState(0);
   const [current, setCurrent] = useState<CurrentSearch>();
   const [sessionId, setSessionId] = useState()
+    const [citations, setCitations] = useState<Citation>([])
```

Comment:
(minor) indentation looks funny on GH, is it aligned?

Reply:
my editor assumes that it should be 4 spaces, not 2. This really should just be linted/autoformatted before pushing, but I haven't got round to it